src/views/Patient/MedReqDropDown/etasuStatus/EtasuStatus.tsx

```diff
@@ -55,11 +55,23 @@ const EtasuStatus = (props: EtasuStatusProps) => {
     const etasuUrl = `${env
       .get('REACT_APP_REMS_ADMIN_SERVER_BASE')
       .asString()}/etasu/met/patient/${patientFirstName}/${patientLastName}/${patientDOB}/drugCode/${drugCode}`;
+
     axios({
       method: 'get',
       url: etasuUrl
     }).then(
       response => {
+        // Sorting an array mutates the data in place.
+        (response.data as RemsMetEtasuResponse).metRequirements.sort(
+          (first: MetRequirements, second: MetRequirements) => {
+            // Keep the other forms unsorted.
+            if (second.requirementName.includes('Patient Status Update')) {
+              // Sort the Patient Status Update forms in descending order of timestamp.
+              return second.requirementName.localeCompare(first.requirementName);
```

Comment:
This is a bit of a strange way to compare timestamps.  Maybe we need a task to get the timestamp as a property of the requirement instead of just tossing it into the name.  Same goes for checking if something is a patient status update by checking its name.  Need a more robust method of checking if something is "sortable".

Reply:
sorting could be done at rems admin, but it should still have a timestamp regardless.  The name should just be the name, no extra information